eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/FieldTypeParameterProviderRegistryPass.php

```diff
@@ -17,6 +17,9 @@
  */
 class FieldTypeParameterProviderRegistryPass implements CompilerPassInterface
 {
+    public const EZPUBLISH_FIELD_TYPE_PARAMETER_PROVIDER = 'ezpublish.fieldType.parameterProvider';
+    public const EZPLATFORM_FIELD_TYPE_PARAMETER_PROVIDER = 'ezplatform.field_type.parameter_provider';
```

Comment:
Why do we need `const`s if their names are exactly the same as their values? I would go with something more meaningful and explicit, like `FIELD_TYPE_PARAMETER_PROVIDER_SERVICE_TAG` and `DEPRECATED_FIELD_TYPE_PARAMETER_PROVIDER_SERVICE_TAG`